crates/subspace-node/src/chain_spec_utils.rs

```diff
@@ -18,6 +20,11 @@ pub(crate) fn chain_spec_properties() -> Properties {
     );
     properties.insert("tokenDecimals".to_string(), DECIMAL_PLACES.into());
     properties.insert("tokenSymbol".to_string(), "tSSC".into());
+    let domains_bootstrap_nodes = Map::<String, Value>::new();
```

Comment:
Would have been nice to use `DomainId` or something like that as a key here rather than a string.

Reply:
With my limited understanding of `serde_json`, it is not possible directly since Map hash impl for only `String`. It is possible to have `DomainId` using https://github.com/jonasbb/serde_with or something similar but I'm not sure if its worth the effort. I would lean towards using as is